cmd/subnetcmd/addValidator.go

```diff
@@ -48,16 +50,16 @@ This command currently only works on subnets deployed to the Fuji testnet.`,
 	}
 	cmd.Flags().StringVarP(&keyName, "key", "k", "", "select the key to use")
 	cmd.Flags().StringVar(&nodeIDStr, "nodeID", "", "set the NodeID of the validator to add")
-	cmd.Flags().StringVar(&weightStr, "weight", "", "set the staking weight of the validator to add")
-	cmd.Flags().StringVar(&startTimeStr, "start-time", startTimeDefault.Format(constants.TimeParseLayout), "start time when this validator starts validating, in 'YYYY-MM-DD HH:MM:SS' format")
-	cmd.Flags().DurationVar(&duration, "staking-period", constants.MaxStakeDuration, "how long this validator will be staking")
+	cmd.Flags().Int64Var(&weight, "weight", 0, "set the staking weight of the validator to add")
```

Comment:
I think you have Uint64Var available

Reply:
I got tripped up, I thought it would be `UInt` not `Uint`. Fixed now.